Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -362,6 +362,41 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 }
 /*endregion*/
 
+/*region bocListValidationRow*/
+div.bocList div.bocListTableBlock tr.bocListValidationRow
+{
+  border-bottom: var(--boclist-tabledata-border);
+  border-top: var(--boclist-tabledata-border);
+  border-top-style: dashed;
+  color: red;
+}
+
+div.bocList table.bocListTable tr.bocListValidationRow.selected td
+{
+  background-color: var(--color-highlight);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow td ul
+{
+  margin: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow div.bocListValidationRowStickyContainer
+{
+  width: 300px;
```

Comment:
I would choose a higher value for the `width` to display all the error text without a scrollbar for now. Once we integrate the width calculation we won't have to specify it but until then its better to have a no scrollbar.

If you want we can add the width detection right now since it should be much work. We already have a `resizeHandler` in line 512 of BocList.ts. We can then use `LayoutUtility.GetWidth(scrollableContainer)` to set the width of the overflow container. This would allow us to get rid of this `width` setting here entirely.

We have discuss with mike what a good cut-off point in terms of size would be. Right now, we have `400px` but I think that is a bit to small - a bigger start for the overflow is probably better.